app/components/primer/layout_component.rb

```diff
@@ -3,6 +3,8 @@
 module Primer
   # Use `Layout` to build a main/sidebar layout.
   class LayoutComponent < Primer::Component
+    status :deprecated
```

Comment:
What do you think about providing a deprecation message suggesting the new component, with a brief description of why it's preferred?

Reply:
we can do that in a follow up PR (to all deprecated components)